.github/workflows/backend-test.yml

```diff
@@ -0,0 +1,27 @@
+name: backend-test
+run-name: ${{ github.actor }} is testing packit backend features
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    branches:
+      - 'main'
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java-version: [ 17 ]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up Java JDK ${{ matrix.java-version }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java-version }}
+          distribution: "temurin"
+          cache: "gradle"
+      - name: test backend
+        working-directory: api
+        run: ./gradlew :app:test --stacktrace
```

Comment:
So now we have
* a shell script for the db
* a `Makefile` for the front-end 
* raw bash commands like this for the back-end

I think we should pick an approach (either `Make` or shell scripts) and use it consistently throughout!

Reply:
Better to get rid of Makefile, the combination is very confusing.